### ISSUE TITLE
Make use of custom colours in KaTeX

### ISSUE THREAD

**Issue by varkor** [varkor/quiver#202]
Currently, `\definecolor` only adds colours to the label colour popup:
<img width="115" alt="image" src="https://github.com/varkor/quiver/assets/3943692/440d7921-ec3a-4349-b087-49efa0a06a93">
It would be useful to also allow colours to be used with commands like `\color` in KaTeX.

This is currently blocked on https://github.com/KaTeX/KaTeX/issues/750.

**Comment by varkor** [comment]
Closing in favour of https://github.com/varkor/quiver/issues/241.
